### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by MarcuzAizen**
Hahaha, usbon sa nakog balik, ,wa jud nagkadimao

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,9 +9,9 @@ LOG_CHANNEL=stack
 DB_CONNECTION=mysql
 DB_HOST=127.0.0.1
 DB_PORT=3306
-DB_DATABASE=laravel
+DB_DATABASE=LVA
 DB_USERNAME=root
-DB_PASSWORD=
+DB_PASSWORD=root
```
Comment (kertnik05) [inline .env.example:14]: you are not supposed to configure the db here in this file. Do that in .env file

Comment (MarcuzAizen) [reply .env.example:14]: Ahh okay

**Comment by kertnik05** [comment]
Also change the PR title into something relevant to what you did

**Comment by kertnik05** [comment]
Also check this out:
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/working-with-forks

**Review by kertnik05 — CHANGES_REQUESTED** [review]
In AcadYearController,, remove all comments, create() & edit() function

**Comment by kertnik05** [comment]
> In AcadYearController,, remove all comments, create() & edit() function

also, the AcadYearController should be in the Controllers/API directory

File: app/Http/Controllers/AcadYearController.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
```
Comment (kertnik05) [inline app/Http/Controllers/AcadYearController.php:3]: API controllers should be in Controllers/API directory

**Review by kertnik05 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/AcadYearController.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        $acad_years = AcadYear::paginate(10);
+        return AcadYearResource::collection($acad_years);
+    }
+
+    public function create()
+    {
+        //return view('acad-year.create');
+        $acadyear = App\Models\AcadYear::create(['start' => date('start')]);
+        $acadyear = App\Models\AcadYear::create(['end' => date('end')]);
+    }
```
Comment (kertnik05) [inline app/Http/Controllers/AcadYearController.php:23]: remove create() function

File: app/Models/AcadYear.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class AcadYear extends Model
+{
+    use HasFactory;
+ 
+    //protected $table = "acad_years";
```
Comment (kertnik05) [inline app/Models/AcadYear.php:12]: remove comment

File: routes/api.php [hunk 5]
```diff
@@ -2,18 +2,15 @@
...
-|
-*/
 
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+
+//Route::put("acad-year", [AcadYearController::class, "update"]);
+Route::apiResource("acad-year", AcadYearController::class);
```
Comment (kertnik05) [inline routes/api.php:16]: url should be in plural form. change url to plural form (acad-year -> acad-years)

File: routes/api.php [hunk 6]
```diff
@@ -2,18 +2,15 @@
...
-| is assigned the "api" middleware group. Enjoy building your API!
-|
-*/
 
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+
+//Route::put("acad-year", [AcadYearController::class, "update"]);
```
Comment (kertnik05) [inline routes/api.php:15]: remove comment

**Review by kertnik05 — CHANGES_REQUESTED** [review]

File: database/migrations/2020_10_28_065725_create_acad_years_table.php [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('acad_years', function (Blueprint $table) {
+            $table->id();
+            $table->date('start')->unique();
+            $table->date('end')->unique();
```
Comment (kertnik05) [inline database/migrations/2020_10_28_065725_create_acad_years_table.php:19]: change data type to `year`

**Review by kertnik05 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**